helm/superset/values.yaml

```diff
@@ -347,6 +347,12 @@ init:
     fi
   ## Annotations to be added to init job pods
   podAnnotations: {}
+  ## Node selector to be added to init job pods
+  nodeSelector: {}
+  ## Tolerations to be added to init job pods
+  tolerations: []  
+  ## Affinity to be added to init job pods
+  affinity: {}
```

Comment:
Could you rebase and update `values.schema.json` too?

https://github.com/apache/superset/blob/4e2bdd49c4f23ace04ccde652fc6dfc8a4a24b0d/helm/superset/values.schema.json#L312-L378